index.js

```diff
@@ -1,7 +1,6 @@
 'use strict';
 var loaderUtils = require('loader-utils');
 var stylus = require('stylus');
-var path = require('path');
```

Comment:
`path` was only used to build-up the `paths` variable, and the `paths` variable isn't used.